plugins/psgi/psgi_loader.c

```diff
@@ -12,7 +12,7 @@ XS(XS_input_seek) {
 	struct wsgi_request *wsgi_req = current_wsgi_req();
 
         psgi_check_args(1);
```

Comment:
What this does is make sure you have at least 1 (argument), i.e. the object (0) and the position.

Now you can just call $input-seek() and it won't error out.

@unbit: The full interface is $input->seek(0, [$whence]), where $whence is SEEK_{SET,CUR,END}. See "perldoc -f seek", any chance of supporting that?

I don't need it for anything, just curious.